From c35ac9502a372308347aedbd2b0381e756df4ccc Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Thu, 2 Aug 2018 17:36:29 +0200
Subject: [PATCH 12/13] LineageParts: Allow brightness control for segmented
 LEDs

* The sdk now supports control of non-RGB segmented battery lights,
  and RGB ones already could support it via scaling RGB values

Change-Id: I834134e82a86d396e77b4eca752559c554609776
---
 .../lineageparts/notificationlight/BatteryLightSettings.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/org/lineageos/lineageparts/notificationlight/BatteryLightSettings.java b/src/org/lineageos/lineageparts/notificationlight/BatteryLightSettings.java
index 9f49d03..2637bef 100644
--- a/src/org/lineageos/lineageparts/notificationlight/BatteryLightSettings.java
+++ b/src/org/lineageos/lineageparts/notificationlight/BatteryLightSettings.java
@@ -148,7 +148,7 @@ public class BatteryLightSettings extends SettingsPreferenceFragment implements
         }
 
         // Remove battery LED brightness controls if we can't support them.
-        if (segmentedBatteryLed || (!mMultiColorLed && !mHALAdjustableBrightness)) {
+        if (!mMultiColorLed && !mHALAdjustableBrightness) {
             prefSet.removePreference(prefSet.findPreference(BRIGHTNESS_SECTION));
         }
 
-- 
2.17.1

